assets/training/finetune_acft_hf_nlp/src/preprocess/preprocess.py

```diff
@@ -205,6 +209,22 @@ def pre_process(parsed_args: Namespace, unparsed_args: list):
                     f"{list(T5_CODE2LANG_MAP.keys())}"
                 ))
             )
+        
+        logger.info("Got model_type = " + model_type)
+
+    # Overriding from_pretrained method for AutoConfig to use trust_remote_code. TODO: Move to model specific config
+    if hasattr(parsed_args, "model_type") and parsed_args.model_type in [MIXFORMER_SEQUENTIAL]:
```

Comment:
Would it be possible to use _ACFT_CONFIG_ instead of overriding the _AutoConfig_ class?